backend/core/src/listings/listings.controller.ts

```diff
@@ -39,9 +40,14 @@ export class ListingsController {
     required: false,
     type: String,
   })
+  @ApiProperty({ name: "filter", required: false })
   @UseInterceptors(CacheInterceptor)
-  public async getAll(@Query("jsonpath") jsonpath?: string): Promise<ListingDto[]> {
-    return mapTo(ListingDto, await this.listingsService.list(jsonpath))
+  @UseInterceptors(FilterInterceptor)
```

Comment:
@seanmalbert Please take a look at how PaginatedApplicationListQueryParams is defined and used in ApplicationsController and lets follow that convention because it's standard for NestJS query params parsing, which is exactly what FilterInterceptor is doing.